alphalens/tears.py

```diff
@@ -428,12 +429,12 @@ def create_turnover_tear_sheet(factor_data, turnover_periods=None):
     """
 
     if turnover_periods is None:
-        input_periods = utils.get_forward_returns_columns(
+        turnover_periods = utils.get_forward_returns_columns(
             factor_data.columns, require_exact_day_multiple=True,
-        )
-        turnover_periods = list(
-            map((lambda x: pd.Timedelta(x).days), input_periods.get_values())
-        )
+        ).get_values()
+    turnover_periods = list(
```

Comment:
Is it intentional that you replace the variable? Looks a bit odd.

Reply:
Yes that was intentional, but agree it's confusing to overwrite it twice. I'll switch back to using the helper variable.